pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -209,50 +207,3 @@ func (r *Reconciler) updateStatus(desired *eventingv1alpha1.KnativeEventing) (*e
 	existing.Status = desired.Status
 	return r.KnativeEventingClientSet.OperatorV1alpha1().KnativeEventings(desired.Namespace).UpdateStatus(existing)
 }
-
-// Delete obsolete resources from previous versions
```

Comment:
why are these changes in here?

Reply:
@k4leung4 This function is used to delete the old resources in the old version. However, this is a fix for the old version 0.12, so I should not have added the resource deletion.